build.md

```diff
@@ -7,15 +7,24 @@ Prerequisites:
 * docker is installed and running
 * wget is installed
 
-After setting the `GOPATH` env var correctly, just run `make <action...>` from the command line, within the same directory where `Makefile` resides. For example `make package clean` will run the `package` and then the `clean` actions.
+After setting the `GOPATH` env var correctly, just run `make <action...>` from the command line, within the same directory where `Makefile` resides. 
+
+For example - if you want to obtain a fully built Appsody CLI for all the supported platform run:
+```
+make clean package
+```
+which will run the `clean` action, and then the `package` action. The latter produces the built binaries (as described in the next section).
+
+If you inspect the `Makefile`, you'll notice that it invokes a number of scripts - and you'll notice a few conditional paths, because certain Linux commands behave differently on OS/X and elsewhere.
```

Comment:
Minor comment - wherever we refer to OS/X in our docs/READMEs we should update it to be macOS

Reply:
Done.